### PR TITLE
Creating the respective condominium project queries

### PR DESCRIPTION
**Pull request by Tejada01DO**
Design of queries without their functions

### CODE REVIEW

File: Client/Pages/Consults/C_Condominiums.razor [hunk 1]
```diff
@@ -0,0 +1,50 @@
+@page "/ConsultCondominius"
```
Comment (JaSsT20) [inline Client/Pages/Consults/C_Condominiums.razor:1]: The name of the URL should be the same as the file name, btw, you missed an "m" letter in the URL.

Should be: @page "/ConsultCondominius" --> @page "/C_Condominiums"

File: Client/Pages/Consults/C_Condominiums.razor [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+<div class="card">
+    <div class="card-header">
+        <h2>Consults of Condominius</h2>
```
Comment (JaSsT20) [inline Client/Pages/Consults/C_Condominiums.razor:5]: Missing one "m" and one "s" is left over: "Consults of Condominius" --> "Consult of condominiums"

**Review by JaSsT20 — CHANGES_REQUESTED** [review]
- Some writing errors need to be corrected.

- Changes have been made that were not requested on your part, next time, make sure you upload only what you were asked to do.

**Review by JaSsT20 — APPROVED** [review]
LGFM

File: Client/Pages/Consults/C_Condominiums.razor (hunk 1 above)
Comment (Tejada01DO) [reply Client/Pages/Consults/C_Condominiums.razor]: Completing in the commit "Resolving spelling and URL name errors"

File: Client/Pages/Consults/C_Condominiums.razor (hunk 2 above)
Comment (Tejada01DO) [reply Client/Pages/Consults/C_Condominiums.razor]: Completing in the commit "Resolving spelling and URL name errors"

**Pull request merged**